### DISCUSSION TITLE
Installation method failing on multiple machines

### DISCUSSION THREAD

**Discussion by sgvdm** [bionic-gpt/bionic-gpt#287] [General]
I have now tried on 3 different Linux boxes (fresh install latest Ubuntu)
Used curl and then docker-compose up and get following error message:

"The compose file ./dockercompose.yml 27 is invalid because: Unsupported config option for services:pipeline-job"

Any one else have this and what is the fix?

**Comment by tbendien** [comment]
The error message refers to a "dockercompose.yml" file

Are you running the following command to bring the system up?

,,,
docker-compose -f docker-compose up
,,,,

You may be missing a hyphen in your command?

**Comment by sgvdm** [comment]
Thank you for your reply. When I run your command I get this error:".FileNotFound: [Errno 2] No such file or directory; './docker-compose'


On Jan 4, 2024, at 2:08 PM, tbendien ***@***.***> wrote:

﻿

The error message refers to a "dockercompose.yml" file

Are you running the following command to bring the system up?

,,,
docker-compose -f docker-compose up
,,,,

You may be missing a hyphen in your command?

**Comment by tbendien** [comment]
It's possible that docker-compose is not installed properly on your host.

If you are using a Debian/Ubuntu based host, can you run:

sudo apt update && sudo apt upgrade -y

then:

sudo apt install docker-compose

then:
docker-compose --version

that will show you which version of Docker-compose you are running.

Likely it will be:
docker-compose version 1.29.2

After you do that, try running the commands to bring the system up:

docker-compose -f docker-compose.yml up

Let me know if that works?

**Comment by sgvdm** [comment]
Thank you for the detailed instructions. I applied these. I verified that I had the most recent version of Docker for Debian.

Unfortunately, when I run the final command to bring up the container - I get the same error message. Interestingly enough, the error is sightly different if I run standard vs using sudo.


ERROR: The Compose file './docker-compose.yml' is invalid because:

Unsupported config option for services: 'embeddings api"


ERROR: The Compose file'. /docker-compose.yml' is invalid because:

Unsupported config option for services:'migration'



________________________________

**Reply by tbendien** [reply]
Ok, I will post some yaml files to install Bionic with the December 2023 version

That may help.

Stay tuned for an update by tomorrow. We'll get you up and running 👍👍😀

**Comment by sgvdm** [comment]
Your help is much appreciated !

On Jan 4, 2024, at 6:16 PM, tbendien ***@***.***> wrote:

﻿

Ok, I will post some yaml files to install Bionic with the December 2023 version

That may help.

Stay tuned for an update by tomorrow. We'll get you up and running 👍👍😀
